src/components/Pagination/Pagination.st.css

```diff
@@ -43,7 +43,7 @@
 }
 
 .pageButtonRules {
-  -st-mixin: buttonTransition;
+  /*-st-mixin: buttonTransition;*/
```

Comment:
??

Reply:
It seems there's an issue with nested mixins like these inside a stylesheet that we partially mixin.
So the Example Pagination component uses `-st-partial-mixin`, and this break the build.

I need to investigate more, it might be a bug with the partial mixins or something on our side.
Didn't get to investigate it yet.

I left this comment out as a reminder